src/main/java/net/sf/jabref/gui/FileExtensions.java

```diff
@@ -12,36 +14,34 @@
 public enum FileExtensions {
 
     //important: No dot before the extension!
-    BIBTEX_DB(
-            String.format("%1s %2s", "BibTex", Localization.lang("Database")), "bib"),
-    BIBTEXML(
-            Localization.lang("%0 file", "BibTeXML"), "bibx"),
-    AUX(
-            Localization.lang("%0 file", "AUX"), "aux"),
-    SILVER_PLATTERN(
-            Localization.lang("%0 file", "SilverPlatter"), "dat"),
-    PUBMED(
-            Localization.lang("%0 file", "PubMed"), "fcgi"),
-    RIS(
-            Localization.lang("%0 file", "RIS"), "ris"),
-    ENDNOTE(
-            Localization.lang("%0 file", "Endnote/Refer"), "ref"),
-    JSTYLE(
-            Localization.lang("Style file"), "jstyle"),
-    LAYOUT(
-            Localization.lang("Custom layout file"), "layout"),
-    XML(
-            Localization.lang("%0 file", "XML"), "xml"),
-    TERMS(
-            Localization.lang("Protected terms file"), "terms"),
-    TXT(
-            Localization.lang("%0 file", Localization.lang("plain text")), "txt"),
-    CLASS(
-            Localization.lang("%0 file", "CLASS"), "class"),
-    JAR(
-            Localization.lang("%0 file", "JAR"), "jar"),
-    ZIP(
-            Localization.lang("%0 file", "ZIP"), "zip");
+    BIBTEX_DB(String.format("%1s %2s", "BibTex", Localization.lang("Database")), "bib"),
+    BIBTEXML(Localization.lang("%0 file", "BibTeXML"), "bibx"),
+    BILBIOSCAPE(Localization.lang("%0 file", "Biblioscape"), "txt"),
+    COPAC(Localization.lang("%0 file", "Copac"), "txt"),
+    ENDNOTE(Localization.lang("%0 file", "Endnote/Refer"), "ref", "enw"),
+    FREECITE(Localization.lang("%0 file", "FreeCite"), "txt", "xml"),
+    INSPEC(Localization.lang("%0 file", "INSPEC"), "txt"),
+    ISI(Localization.lang("%0 file", "ISI"), "isi", "txt"),
+    MEDLINE(Localization.lang("%0 file", "Medline"), "nbib", "xml"),
+    MEDLINE_PLAIN(Localization.lang("%0 file", "MedlinePlain"), "nbib", "txt"),
+    MSBIB(Localization.lang("%0 file", "MSBib"), "xml"),
+    OVID(Localization.lang("%0 file", "Ovid"), "txt"),
+    PDF_CONTENT(Localization.lang("%0 file", "PDF content "), "pdf"),
+    PUBMED(Localization.lang("%0 file", "PubMed"), "fcgi"),
+    REPEC("REPEC New Economic Papers (NEP)", "txt"),
+    RIS(Localization.lang("%0 file", "RIS"), "ris"),
+    SILVER_PLATTER(Localization.lang("%0 file", "SilverPlatter"), "dat", "txt"),
+    XMP(Localization.lang("XMP-annotated PDF"), "pdf"),
+
+    AUX(Localization.lang("%0 file", "AUX"), "aux"),
+    JSTYLE(Localization.lang("Style file"), "jstyle"),
+    LAYOUT(Localization.lang("Custom layout file"), "layout"),
+    TERMS(Localization.lang("Protected terms file"), "terms"),
+    TXT(Localization.lang("%0 file", Localization.lang("plain text")), "txt"),
```

Comment:
Or maybe even go for "Plain text file" now that I read the whole line of code. :-)